CryptoPkg/Library/BaseCryptLibMbedTls/Pk/CryptRsaPss.c

```diff
@@ -75,78 +75,24 @@ RsaPssVerify (
 
   switch (DigestLen) {
     case SHA256_DIGEST_SIZE:
-      md_alg     = MBEDTLS_MD_SHA256;
-      ShaCtxSize = Sha256GetContextSize ();
-      ShaCtx     = AllocateZeroPool (ShaCtxSize);
-
-      Status = Sha256Init (ShaCtx);
-      if (!Status) {
-        return FALSE;
-      }
-
-      Status = Sha256Update (ShaCtx, Message, MsgSize);
-      if (!Status) {
-        FreePool (ShaCtx);
-        return FALSE;
-      }
-
-      Status = Sha256Final (ShaCtx, HashValue);
-      if (!Status) {
-        FreePool (ShaCtx);
+      MdAlg     = MBEDTLS_MD_SHA256;
+      if (mbedtls_sha256_ret (Message, MsgSize, HashValue, FALSE) != 0) {

```

Comment:
mbedtls_sha*_ret() are deprecated API of MbedTls2.x, please use mbedtls_sha*() of MbedTls3.x